### ISSUE TITLE
New workspaces should be created in their own directory

### ISSUE THREAD

**Issue by KyleW** [nrwl/nx#549]
When using ng-cli without nx the command from projects/ :
```
ng new my-app
```
creates a new directory projects/my-app/ and creates an angular project inside that directory

When using ng-cli with nx:
```
ng new myworkspacename --collection=@nrwl/schematics
```
Creates an angular workspace in the current directory(projects/)

nx should probably follow the same patterns as ng-cli or, at the least, make it clear that you should create a directory first in the docs.

**Comment by KyleW** [comment]
Nevermind. I can't reproduce. Either this was already fixed or I did something funny when I saw the error.

**Comment by FrozenPandaz** [comment]
Pre-emptive Thanks to @Brocco for fixing this.

You can follow angular/angular-cli#549 for updates

**Comment by FrozenPandaz** [comment]
This has been fixed in Angular CLI 6.1.0-rc.0.
https://github.com/angular/angular-cli/releases/tag/v6.1.0-rc.0
